oscovida/oscovida.py

```diff
@@ -746,7 +746,9 @@ def plot_time_step(ax, series, style="-", labels=None, logscale=True):
     return ax
 
 
-def compute_daily_change(series):
+def compute_daily_change(series: pd.Series) -> Tuple[Tuple[pd.Series, str],
+                                                     Tuple[pd.Series, str],
+                                                     Tuple[pd.Series, str]]:
```

Comment:
In this case this is a bit unnecessary as the return type is not re-used at all, but in general I think a neater way to do this is:

```python
import typing

DailyChangeResult = typing.NamedTuple("DailyChange", [
    ('change', Tuple[pd.Series, str]),
    ('smooth', Tuple[pd.Series, str]),
    ('smooth2', Tuple[pd.Series, str])
])

def compute_daily_change(series: pd.Series) -> DailyChangeResult
```

This annotates the types as you already did and also attaches labels to it, which is quite useful.